Microsoft.Toolkit.Uwp.UI.Controls/TextBoxMask/TextBoxMask.cs

```diff
@@ -119,10 +120,38 @@ private static void Textbox_Loaded(object sender, RoutedEventArgs e)
             textbox.TextChanging += Textbox_TextChanging;
             textbox.SelectionChanged += Textbox_SelectionChanged;
             textbox.Paste += Textbox_Paste;
+            textbox.GotFocus += Textbox_GotFocus;
             textbox.SetValue(OldTextProperty, textbox.Text);
             textbox.SelectionStart = 0;
         }
 
+        private static void Textbox_GotFocus(object sender, RoutedEventArgs e)
+        {
+            var textbox = sender as TextBox;
```

Comment:
actually I need to ask @deltakosh is there is any chance I can get a null here from the framework ?